src/hotspot/share/gc/g1/g1CardSetContainers.hpp

```diff
@@ -26,16 +26,14 @@
 #define SHARE_GC_G1_G1CARDSETCONTAINERS_HPP
 
 #include "gc/g1/g1CardSet.hpp"
+#include "logging/log.hpp"
 #include "memory/allocation.hpp"
 #include "runtime/atomic.hpp"
+#include "runtime/thread.hpp"
```

Comment:
It's not clear why some of these #includes are here.  log, thread, spinYield all seem unused in this .hpp.